examples/mplot3d/trisurf3d_demo.py

```diff
@@ -27,6 +27,6 @@
 fig = plt.figure()
 ax = fig.gca(projection='3d')
 
-ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2)
+ax.plot_trisurf(x, y, z, cmap=cm.jet, linewidth=0.2, antialiased=True)
```

Comment:
That's not a change in behavior.  pcolormesh has had no borders and antialiased=False for a long time.  I don't think it makes sense to change that.  The change here is just that it happens to look better with the new colormap *on this specific data* so why not?